### PR TITLE
Eui 6408  select to copy user details

### PR DESCRIPTION
**Pull request by aamirhm**
Resolves # . (This is applicable only if this pull request relates to a GitHub issue, delete the line otherwise)

Notes:
*
*
*

### CODE REVIEW

**Review by DavidJayakumar — APPROVED** [review]
LGTM

File: projects/exui-common-lib/src/lib/models/filter.model.ts [hunk 1]
```diff
@@ -21,6 +22,8 @@ export interface FilterConfig {
   cancelSetting?: FilterSetting;
   showCancelFilterButton?: boolean;
   cancelButtonCallback?(): void;
+  applyButtonCallback?(): void;
+  copyFields?(form: FormGroup): FormGroup;
```
Comment (udaydenduluri33) [inline projects/exui-common-lib/src/lib/models/filter.model.ts:26]: @aamirhm  does this need to be an optional function in the parameter?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hmcts/rpx-xui-common-lib",
-  "version": "1.7.49-update-from-r3.5-to-4-v3",
+  "version": "1.7.47-callback-copy-user-v1",
```
Comment (connorpgpmcelroy) [inline package.json:3]: Preferable to have same version as base branch version but different tag

File: projects/exui-common-lib/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hmcts/rpx-xui-common-lib",
-  "version": "1.7.49-update-from-r3.5-to-4-v3",
+  "version": "1.7.47-callback-copy-user-v1",
```
Comment (connorpgpmcelroy) [inline projects/exui-common-lib/package.json:3]: Above applies

File: projects/exui-common-lib/src/lib/models/filter.model.ts (hunk 1 above)
Comment (aamirhm) [reply projects/exui-common-lib/src/lib/models/filter.model.ts:26]: Yes, it is an optional function as only the user form needs a `copy` profile button.

File: package.json (hunk 2 above)
Comment (aamirhm) [reply package.json]: Updated.

File: projects/exui-common-lib/package.json (hunk 3 above)
Comment (aamirhm) [reply projects/exui-common-lib/package.json]: Updated.

**Review by udaydenduluri33 — APPROVED** [review]

**Review by connorpgpmcelroy — APPROVED** [review]
LGTM

**Pull request merged**